src/Inkluzitron/Modules/InviteModule.cs

```diff
@@ -0,0 +1,111 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Discord;
+using Discord.Commands;
+using Discord.WebSocket;
+using Inkluzitron.Data;
+using Inkluzitron.Data.Entities;
+using Inkluzitron.Extensions;
+using Inkluzitron.Services;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+
+namespace Inkluzitron.Modules
+{
+    [Name("Invites")]
+    [Group("invite")]
+    [Summary("Vytvoří jednorázový invite link, nebo zobrazí kým byl uživatel pozván.")]
+    public class InviteModule : ModuleBase
+    {
+        private IConfiguration Config { get; }
+        private DiscordSocketClient DiscordSocketClient { get; }
+
+        private BotDatabaseContext DbContext { get; set; }
+        private DatabaseFactory DatabaseFactory { get; }
+        private UsersService UsersService { get; }
+
+        public InviteModule(IConfiguration config,
+            DatabaseFactory databaseFactory,
+            UsersService usersService,
+            DiscordSocketClient discordSocketClient)
+        {
+            Config = config;
+            DatabaseFactory = databaseFactory;
+            UsersService = usersService;
+            DiscordSocketClient = discordSocketClient;
+        }
+
+        override protected void BeforeExecute(CommandInfo command)
+        {
+            DbContext = DatabaseFactory.Create();
+            base.BeforeExecute(command);
+        }
+
+        override protected void AfterExecute(CommandInfo command)
+        {
+            DbContext?.Dispose();
+            base.AfterExecute(command);
+        }
+
+        [Command("")]
+        [Summary("Vytvoří jednorázový invite link a pošle ho do DM.")]
+        [RequireBotPermission(GuildPermission.CreateInstantInvite)]
+        public async Task CreateInviteAsync()
+        {
+            var invite = await Context.Guild.DefaultChannel.CreateInviteAsync(
+                null,
+                2, // create 2 uses for invite because if only 1 usage is available it's automatically deleted after joining
+                false,
+                true,
+                new RequestOptions()
+                {
+                    AuditLogReason = $"User with ID {Context.User.Id} and name {Context.User.Username} wants to invite someone!"
+                });
+
+            var user = await DbContext.GetOrCreateUserEntityAsync(
+                Context.Message.Author);
+
+            var inviteDb = new Invite
```

Comment:
Rád bych, kdyby byl implementovaný nějaký maximální limit, kolik aktivních invites může mít jeden uživatel.

Reply:
Myslis, ze by se nasel nekdo, kdo by vytvoril max. pocet invitu na server?

Reply:
Je to edge case. Rád bych to viděl, ale nutný to určitě není. Uvidíme časem jestli toho někdo zneužije, ale zatím bych to teda nechal tak, jak to je.